client/my-sites/customer-home/cards/tasks/go-mobile/index.js

```diff
@@ -13,33 +13,19 @@ import { preventWidows } from 'lib/formatting';
 import AppsBadge from 'blocks/get-apps/apps-badge';
 import { TASK_GO_MOBILE_ANDROID, TASK_GO_MOBILE_IOS } from 'my-sites/customer-home/cards/constants';
 
-/**
- * Image dependencies
- */
-import appleStoreLogo from 'assets/images/customer-home/apple-store.png';
-import googlePlayLogo from 'assets/images/customer-home/google-play.png';
-
 const GoMobile = ( { isIos } ) => {
 	const translate = useTranslate();
 
 	const actionButton = isIos ? (
 		<AppsBadge
 			storeLink="https://apps.apple.com/app/apple-store/id335703880?pt=299112&ct=calypso-customer-home&mt=8"
```

Comment:
Seems this is the same link as the default set in `AppsBage` so I think we can drop it:

```suggestion
			referrer="calypso-customer-home"
```

Reply:
sure.